lib/tasks/pipeline_monitor.rake

```diff
@@ -3,6 +3,17 @@ require 'English'
 require 'json'
 require 'thread/pool'
 
+Rails.application.config.after_initialize do
+  ActiveRecord::Base.connection_pool.disconnect!
+
+  ActiveSupport.on_load(:active_record) do
+    config = ActiveRecord::Base.configurations[Rails.env] ||
+              Rails.application.config.database_configuration[Rails.env]
+    config['pool'] = 20
```

Comment:
Is this to match the THREAD_COUNT? Should we use the same constant?

Reply:
Good catch thanks